scripts/superdesk-search/services/TagService.js

```diff
@@ -23,6 +23,13 @@ export function TagService($location, desks, userList, metadata, search) {
 
     var cvs = search.cvs;
 
+    function tag(label, value) {
+        return {
+            label: label,
+            value: value || label
+        };
+    }
+
     function initSelectedParameters (parameters) {
         tags.selectedParameters = [];
```

Comment:
I think the only advantage is better encapsulation, since `tag` is only used in `initSelectedParameters`, defining it there makes this obvious. It's just styling really... Not necessarily something that needs changing, just a personal opinion.